app/views/donors/show.html.slim

```diff
@@ -3,9 +3,12 @@
 .row
   #edit_partial
 
-- if @new_donation == "true"
+- if @new_donation == "true" && @donor.name != "Anonymous"
   .row
     = render "edit"
+- elsif @donor.name == "Anonymous"
```

Comment:
Can we encapsulate the logic to check whether a user is anonymous or not into a method, `User#anonymous?`?

This was, if we change to an `enum` or similar, we can just change that one method. 😀